packages/client/hmi-client/src/components/workflow/ops/document/tera-document-operation-node.vue

```diff
@@ -128,6 +161,24 @@ watch(
 	},
 	{ immediate: true }
 );
+
+watch(
+	() => document.value?.thumbnail,
+	() => {
+		if (document.value?.thumbnail?.length) {
+			thumbnail.value = `data:image/png;base64, ${document.value.thumbnail}`;
+		}
+	}
+);
```

Comment:
i think you can just add this if statement to the above watcher and things will be the same